duckduckgo/duckduckgo-privacy-extension
# Remove unused options file

**Reviewer:** @dharb 

## Description:

**Note** this does not merge into develop.

This PR is removing a file that isn't really used anywhere, so we don't have to worry about browserifying it.

We do have an options page, which loads a file called `public/js/options.js`: https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/develop/shared/html/options.html#L69.

However, when you look at the Gruntfile you'll see that `public/js/options.js` is generated from the options file in the UI folder: https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/develop/Gruntfile.js#L25

Plus, it's got loads of options we don't really seem to support anymore.

## Steps to test this PR:

1. Build the extension (FF and Chrome)
2. Check that the options page renders and works as before


## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
